internal/gengapic/genrest.go

```diff
@@ -382,7 +382,7 @@ func (g *generator) generateURLString(m *descriptor.MethodDescriptorProto) error
 	fmtStr := info.url
 	// TODO(dovs): handle more complex path urls involving = and *,
 	// e.g. v1beta1/repeat/{info.f_string=first/*}/{info.f_child.f_string=second/**}:pathtrailingresource
-	re := regexp.MustCompile(`{([a-zA-Z0-9_.]+?)}`)
+	re := regexp.MustCompile(`{([a-zA-Z0-9_.]+?)(=[^}]+)?}`)
```

Comment:
yeah this would capture the `=` in the second capture group, but idk if we even need that value tbh so not a big deal.

Reply:
I think we'd also want to disallow `{` to ensure there are no nested variables. Also not a big deal.

Reply:
To your first comment, the parens that included the `=` were just grouping for the `?`... I think we'll leave this?

To the second, ok, I'll add `{`.

Reply:
Done.